toDoApp/src/screens/Details/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { SafeAreaView, View } from 'react-native'
+import { styles } from './styles'
+import { TextRectangle } from './TextRectangle'
+import { RouteProp } from '@react-navigation/native'
+import { StackNavigationProp } from '@react-navigation/stack'
+import { RootStackParamList } from '../../navigation/types'
+
+type DetailsScreenNavigationProp = StackNavigationProp<
+  RootStackParamList,
+  'Details'
+>
+
+type DetailsScreenRouteProp = RouteProp<RootStackParamList, 'Details'>
+
+type Props = {
+  route: DetailsScreenRouteProp
+  navigation: DetailsScreenNavigationProp
+}
```

Comment:
This is what you should do instead of the above in the navigation types. Also this go inside a types file in the scene folder